package.json

```diff
@@ -36,13 +36,16 @@
   "browser": "./dist/otpauth.esm.js",
   "exports": {
     ".": {
-      "types": "./dist/otpauth.d.ts",
       "bun": "./dist/otpauth.esm.js",
       "deno": "./dist/otpauth.esm.js",
       "node": {
         "import": "./dist/otpauth.node.mjs",
-        "require": "./dist/otpauth.node.cjs"
+        "require": {
+          "types": "./dist/otpauth.d.cts",
+          "default": "./dist/otpauth.node.cjs"
+        }
       },
+      "types": "./dist/otpauth.d.ts",
       "default": "./dist/otpauth.esm.js"
```

Comment:
Happy to rearrange it :+1:. You can also remove the `bun` and `deno` entries as they will just use `default` instead, shall I do that while I'm there?

> You would also need to define the ./dist/otpauth.d.cts entry in ./dist/*.

Not sure what you mean by this, do you mean adding the file? I forgot to commit it originally but added it in https://github.com/hectorm/otpauth/pull/447/commits/f5d190e4e4eddc9ace000f75047dc0f9b5e3dad8

Reply:
> Happy to rearrange it 👍. You can also remove the `bun` and `deno` entries as they will just use `default` instead, shall I do that while I'm there?

I prefer to reevaluate the need for `bun` and `deno` entries another time.

> Not sure what you mean by this, do you mean adding the file? I forgot to commit it originally but added it in https://github.com/hectorm/otpauth/commit/f5d190e4e4eddc9ace000f75047dc0f9b5e3dad8

Sorry if I was not clear, I meant that it would also be necessary to update the `types` field of `./dist/*` in the `exports` field, as seen in my initial suggestion.

```json
"./dist/*": {
  "types": {
    "import": "./dist/otpauth.d.ts",
    "require": "./dist/otpauth.d.cts"
  },
  "default": "./dist/*"
}
```